pom.xml

```diff
@@ -198,12 +248,14 @@
         <module>cas-client-integration-jboss</module>
         <module>cas-client-support-distributed-ehcache</module>
         <module>cas-client-support-distributed-memcached</module>
+        <module>cas-client-support-saml</module>
```

Comment:
ah -- I see you did add one.  Any reason that Joda time needs to be scoped beyond this?